Common++/src/SystemUtils.cpp

```diff
@@ -183,18 +185,29 @@ void createCoreVectorFromCoreMask(CoreMask coreMask, std::vector<SystemCore>& re
 	}
 }
 
-std::string executeShellCommand(const std::string &command)
+namespace
```

Comment:
can't we just put `PcloseDeleter` outside?

Reply:
Define outside? Outside the anonymous namespace?

Reply:
yes, any reason that you need the anonymous namespace here?

Reply:
Well, mostly to have the helper functions and structs local to the translation unit. Although I guess it would be better to pull the unnamed namespace up to the global level.

Reply:
Pulled the structure into a top level unnamed namespace (c285b6f).